pipelines/models/tensorflow-facedetection/README.md

```diff
@@ -0,0 +1,25 @@
+# Tensorflow Face Detection
+## Description
+Face detector based on SqueezeNet light (half-channels) as a backbone with a single SSD for indoor/outdoor scenes shot by a front-facing camera. The backbone consists of fire modules to reduce the number of computations. The single SSD head from 1/16 scale feature map has nine clustered prior boxes.
+
+## Test
+### Run OVMS with the model inside
+Execute the following command from the [pipelines](../../) folder:
+```
+docker run -d -u $(id -u):$(id -g) --rm -v ${PWD}/models:/model:Z -p 9000:9000 openvino/model_server:latest --model_name face-detection --model_path /model/tensorflow-facedetection --port 9000 --shape auto
+```
+
+### Test call
+
+```
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/demos/face_detection/python
+pip install -r ../../common/python/requirements.txt
+# In case of errors remove the tensorflow-serving-api version from the ../../common/python/requirements.txt
```

Comment:
Removing tensorflow-serving-api does not work for me. What I had to do to fix the errors is the following:
```
# Patch the requirements file, because tensorflow 2.11.0 is not working
sed -i 's/2.11.0/2.13.0/' client_requirements.txt
```

It, simply, uses tensorflow 2.13.0 instead of 2.11.0. Looks like 2.11 is now broken.
You may want to add that line before `pip install`.

Reply:
Added